dask_saturn/core.py

```diff
@@ -61,7 +66,7 @@ def reset(
         Destroy existing Dask cluster attached to the Jupyter Notebook or
         Custom Deployment and recreate it with the given configuration.
         """
-        print(f"Resetting cluster.")
+        log.info(f"Resetting cluster.")
```

Comment:
```suggestion
        log.info("Resetting cluster.")
```

I think you can remove this `f` (I cleaned it up in #19 so you'll probably get a merge conflict, sorry :grimacing: )